subsys/net/l2/wifi/Kconfig

```diff
@@ -27,3 +27,9 @@ config WIFI_MGMT_RAW_SCAN_RESULTS_ONLY
 	  to the application.
 
 endif # WIFI_MGMT_RAW_SCAN_RESULTS
+
+config WIFI_MGMT_SCAN_MODE_PASSIVE
+	bool "Use Passive scan mode"
+	default y
```

Comment:
Default should be active scan as passive is only used to handle regulatory restrictions, any specific reason for default passive? And also this changes the current behaviour completely.

[Edit: Looking at code below this only means, PASSIVE scan support]

Reply:
Add support to force scan in passive-only mode. Currently, this support is already present via wifi utils. We need to add Kconfig option to set the default mode to passive and provide UMAC API to use to change this at run time if needed. 
For all the below comments, Please check
https://nordicsemi.atlassian.net/browse/SHEL-1628 & 
https://nordicsemi.atlassian.net/browse/SHEL-1568

Reply:
The Kconfig in `wifi_shell` isn't useful, I guess the Kconfig override should be in the driver to always do passive scan e.g., for regulatory tests